trackintel/analysis/tracking_quality.py

```diff
@@ -40,17 +40,33 @@ def temporal_tracking_quality(source, granularity="all", max_iter=60):
     datamodels and their combinations (e.g., staypoints and triplegs sequence).
 
     The temporal tracking quality is the ratio of tracking time and the total time extent. It is
-    calculated and returned per-user in the defined ``granularity``. The possible time extents of
-    the different granularities are different:
+    calculated and returned per-user in the defined ``granularity``. The time extents
+    and the columns for the returned ``quality`` df for different ``granularity`` are:
 
-    - ``all`` considers the time between the latest "finished_at" and the earliest "started_at";
-    - ``week`` considers the whole week (604800 sec)
-    - ``day`` and ``weekday`` consider the whole day (86400 sec)
-    - ``hour`` considers the whole hour (3600 sec).
+    - ``all``:
 
-    The tracking quality of each user is calculated based on his or her own tracking extent.
-    For granularity = ``day`` or ``week``, the quality["day"] or quality["week"] column displays the
-    time relative to the first record in the entire dataset.
+        - time extent: between the latest "finished_at" and the earliest "started_at" for each user.
+        - columns: ``['user_id', 'quality']``.
+
+    - ``week``:
+
+        - time extent: the whole week (604800 sec) for each user.
+        - columns: ``['user_id', 'week_monday', 'quality']``.
+
+    - ``day``:
+
+        - time extent: the whole day (86400 sec) for each user
+        - columns: ``['user_id', 'day', 'quality']``
+
+    - ``weekday``
+
+        - time extent: the whole day (86400 sec) * number of tracked weeks for each user for each user
+        - columns: ``['user_id', 'weekday', 'quality']``
+
+    - ``hour``:
+
+        - time extent: the whole hour (3600 sec) * number of tracked days for each user
+        - columns: ``['user_id', 'hour', 'quality']``
 
```

Comment:
+1 to the docstring update, this summarises everything.
Also, since you did not change the order of the `quality` column, other test functions need not be updated as well (for example, I had to make it -2 at some places instead of -1). Now it can be -1 throughout. 

I think this is a better approach than what I was doing (returning both items: relative and absolute.)